docs/guides/simulate-with-qiskit-sdk-primitives.ipynb

```diff
@@ -9,13 +9,15 @@
    ]
   },
   {
-   "cell_type": "markdown",
-   "id": "26c2809e-c8b4-4e25-996e-c41f72723b1b",
+   "cell_type": "code",
```

Comment:
Do you know why this change happened? `version-info` should be a markdown cell

Reply:
Yes, my bad - I accidentally deleted it, then let it auto-detect the type when I added it back. I've now changed it to markdown